[2024-11-10T19:20:17.140+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-10T19:20:17.181+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline.fetch_data_from_csv manual__2024-11-10T19:20:15.270318+00:00 [queued]>
[2024-11-10T19:20:17.191+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline.fetch_data_from_csv manual__2024-11-10T19:20:15.270318+00:00 [queued]>
[2024-11-10T19:20:17.192+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-11-10T19:20:17.214+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): fetch_data_from_csv> on 2024-11-10 19:20:15.270318+00:00
[2024-11-10T19:20:17.221+0000] {standard_task_runner.py:64} INFO - Started process 191 to run task
[2024-11-10T19:20:17.226+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipeline', 'fetch_data_from_csv', 'manual__2024-11-10T19:20:15.270318+00:00', '--job-id', '600', '--raw', '--subdir', 'DAGS_FOLDER/pipline.py', '--cfg-path', '/tmp/tmph1ac9efb']
[2024-11-10T19:20:17.231+0000] {standard_task_runner.py:91} INFO - Job 600: Subtask fetch_data_from_csv
[2024-11-10T19:20:17.294+0000] {task_command.py:426} INFO - Running <TaskInstance: pipeline.fetch_data_from_csv manual__2024-11-10T19:20:15.270318+00:00 [running]> on host df1fa53d9d8d
[2024-11-10T19:20:17.412+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pipeline' AIRFLOW_CTX_TASK_ID='fetch_data_from_csv' AIRFLOW_CTX_EXECUTION_DATE='2024-11-10T19:20:15.270318+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-10T19:20:15.270318+00:00'
[2024-11-10T19:20:17.413+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-10T19:20:17.414+0000] {logging_mixin.py:188} INFO - Error reading CSV file: [Errno 2] No such file or directory: 'C:/Users/mohammed/Desktop/Airflow/Data-Engineering-Pipeline/***/data_set/journals.csv'
[2024-11-10T19:20:17.415+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-11-10T19:20:17.416+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-10T19:20:17.427+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipeline, task_id=fetch_data_from_csv, run_id=manual__2024-11-10T19:20:15.270318+00:00, execution_date=20241110T192015, start_date=20241110T192017, end_date=20241110T192017
[2024-11-10T19:20:17.479+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-11-10T19:20:17.496+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-10T19:20:17.498+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
